Remotion/Core/Collections.DataStore.UnitTests/LockingDataStoreDecoratorTest.cs

```diff
@@ -80,7 +82,18 @@ public void Get_Value ()
     [Test]
     public void Set_Value ()
     {
-      _helper.ExpectSynchronizedDelegation (store => store["c"] = 17);
+      var indexer = typeof (IDataStore<string, int>).GetProperty ("Item");
```

Comment:
Could you check if `MemberInfoFromExpressionUtility.GetProperty((....ds)=> ds[null] = null) would also work?

Reply:
hmm no, it says `An expression tree may not contain an assignment operator`, so it doesn't work. I don't think there is an easy way to solve this.

Reply:
Okay, tried the getter-version, doesn't work either. Meh...